### ISSUE TITLE
`pixi init` broken with incomplete `pyproject.toml`

### ISSUE THREAD

**Issue by pavelzw** [prefix-dev/pixi#1207]
```shell
❯ cat pyproject.toml
[build-system]
requires = ["setuptools", "wheel"]

❯ pixi init
  × failed to parse project manifest
   ╭─[pyproject.toml:1:1]
 1 │ [build-system]
   · ┬
   · ╰── Missing field `project`
 2 │ requires = ["setuptools", "wheel"]
   ╰────
```

I personally would prefer to still be able to use the `pixi.toml` workflow and not be forced to use `pyproject.toml` as the `pyproject.toml` workflow always(?) introduces `[pypi-dependencies]` as this results in other issues (#1046). Maybe a prompt asking whether pixi should init into `pyproject.toml` or `pixi.toml`?

**Comment by olivier-lacroix** [comment]
Another option would be to gate the pyproject.toml init behind the —pyproject flag

**Comment by ruben-arts** [comment]
I guess that would makes sense, or spawn a interactive choice

**Comment by olivier-lacroix** [comment]
I believe we can close this issue, as pixi init is no more broken with incomplete pyproject.toml :-)

And the choice between pixi.toml / pyproject.toml on init is tracked in https://github.com/prefix-dev/pixi/issues/1615

**Comment by ruben-arts** [comment]
Fixed by #1640 

Now you can do `pixi init --format pixi`
But in interactive mode `pixi init` on a folder with a `pyproject.toml` you'll be asked to choose if you want to add to the pyproject or not.
